sparkpost.class.php

```diff
@@ -69,20 +77,20 @@ public function add_settings_link($links)
 
     public function set_from_name($name)
     {
-        if (!empty($this->options['from_name'])) {
-            return $this->options['from_name'];
-        } else {
-            return $name;
+        if (!empty($this->settings['from_name'])) {
+            $name = $this->settings['from_name'];
         }
+
+        return apply_filters('wpsp_sender_name', $name);
     }
 
     public function set_from_email($email)
     {
-        if (!empty($this->options['from_email'])) {
-            return $this->options['from_email'];
-        } else {
-            return $email;
+        if (!empty($this->settings['from_email'])) {
+            $email = $this->settings['from_email'];
         }
+
+        return apply_filters('wpsp_sender_email', $email);
```

Comment:
same as https://github.com/SparkPost/wordpress-sparkpost/pull/64/files#r83320192

i guess we don't actually need them